test/object-expressions/property-name.js

```diff
@@ -21,5 +21,21 @@ describe("Parser", function () {
   describe("property name", function () {
     expect(ShiftParser.default("({0x0:0})").body.statements[0].expression.properties[0].name.value).to.be("0");
     expect(ShiftParser.default("({2e308:0})").body.statements[0].expression.properties[0].name.value).to.be("" + 1 / 0);
+
+    expect(ShiftParser.default("({get b() {}})").body.statements[0].expression.properties[0].name.value).to.be("b");
+    expect(ShiftParser.default("({set c(x) {}})").body.statements[0].expression.properties[0].name.value).to.be("c");
+
+    expect(ShiftParser.default("({__proto__:0})").body.statements[0].expression.properties[0].name.value).to.be("__proto__");
+
+    expect(ShiftParser.default("({get __proto__() {}})").body.statements[0].expression.properties[0].type).to.be("Getter");
+    expect(ShiftParser.default("({get __proto__() {}})").body.statements[0].expression.properties[0].name.value).to.be("__proto__");
+
```

Comment:
Instead of doubling up each of these tests, let's just compare the whole structure like we do in tests elsewhere. I can describe better in person.